Storage/SQL/BrowserSchema.swift

```diff
@@ -379,6 +381,15 @@ open class BrowserSchema: Schema {
         )
         """
 
+    let faviconSiteURLsCreate = """
+        CREATE TABLE favicon_site_urls (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            site_url TEXT NOT NULL,
```

Comment:
nvm, UNIQUE takes care of that

Reply:
I think this is what we want. Rnewman made a case for why it should continue to be like this.

Reply:
For one, I think this should only be considered as a band-aid for favicons until we move to the Rust Places backend for history. So, I would prefer to not open a can of worms by doing a domain-based lookup here when we do a site-based lookup everywhere else. Also remember that we will only lookup in this table for sites that we haven't collected a favicon for yet in our local history. As soon as you visit the site, we store the favicon the normal way and the bookmarks will also look up through the same thing.

Regarding the index, we already have an index here since we specified uniqueness with that field.

Reply:
On second thought, if we really do want to move towards a domain-based lookup, we could probably refine this schema a little bit more and lay the groundwork for it.

Reply:
Sorry for the latency in my comments. Ok, we'll leave this as-is.